EIPS/eip-1024.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 1024
+title: Add web3.eth.encrypt and web3.eth.decrypt functions
+author: Tope Alabi <alabitemitope@gmail.com>
+status: Draft
+type: Interface Track
```

Comment:
There is no 'interface track'. Please use a type and category from EIP 0.

Reply:
Thanks!

Reply:
This is still not fixed.

Reply:
```suggestion
type: Standards Track
category: Interface
```